yazi-core/src/external/jq.rs

```diff
@@ -1,16 +1,16 @@
 use std::{path::Path, process::Stdio};
 
 use anyhow::Result;
-use tokio::{io::{AsyncBufReadExt, BufReader}, process::Command};
+use tokio::{io::{AsyncBufReadExt, AsyncReadExt, BufReader}, process::Command, select};
 use yazi_config::PREVIEW;
 use yazi_shared::PeekError;
 
 pub async fn jq(path: &Path, skip: usize, limit: usize) -> Result<String, PeekError> {
 	let mut child = Command::new("jq")
-		.args(["-C", "--indent", &PREVIEW.tab_size.to_string(), "."])
+		.args(["-C", "--indent", "-1", "."])
```

Comment:
`--indent -1` can be replaced with a more explicit [`--tab`](https://github.com/jqlang/jq/blob/jq-1.7/src/main.c#L90)

Reply:
Updated!